app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -641,6 +641,17 @@
     <div class="field">{{i18n "user.invited.days_visited"}}</div>
     <div class="value">{{html-safe model.days_visited}}</div>
   </div>
+  <div class="display-row post-edits-count">
+    <div class="field">{{i18n "user.invited.post_edits_count" }}</div>
+    <div class="value">
+      {{html-safe model.post_edits_count}}
+      {{d-button class="widget-button btn-flat heatmap-high btn-icon"
```

Comment:
I think we need to  handle the case where `model.post_edits_count` is `0`. We do not have to display the button if the user has not edited any posts.